docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -282,36 +396,73 @@ When you define an aggregation in a {dfeed}, it must have the following form:
 ----------------------------------
 // NOTCONSOLE
 
+When using a `composite` aggregation:
+
+[source,js]
+----------------------------------
+"aggregations": {
+  "composite_agg": {
+    "sources": [
+      {
+        "date_histogram_agg": {
+          "field": "time",
+          ...settings...
+        }
+      },
+      ...other valid sources...
+      ],
+      ...composite agg settings...,
+      "aggregations": {
+        "timestamp": {
+            "max": {
+              "field": "time"
+            }
+          },
+          ...other aggregations...
+          [
+            [,"aggregations" : {
+              [<sub_aggregation>]+
+            } ]
+          }]
+      }
+   }
+}
+----------------------------------
+// NOTCONSOLE
+
 The top level aggregation must be either a
```

Comment:
Since there are more than two options now, we can't use "either". Perhaps it's clearer now as a list:
```suggestion
The top level aggregation must be one of the following:
```